common/network-common/src/main/java/org/apache/spark/network/server/ChunkFetchRequestHandler.java

```diff
@@ -88,12 +88,14 @@ public void processFetchRequest(
       logger.trace("Received req from {} to fetch block {}", getRemoteAddress(channel),
         msg.streamChunkId);
     }
-    long chunksBeingTransferred = streamManager.chunksBeingTransferred();
-    if (chunksBeingTransferred >= maxChunksBeingTransferred) {
-      logger.warn("The number of chunks being transferred {} is above {}, close the connection.",
-        chunksBeingTransferred, maxChunksBeingTransferred);
-      channel.close();
-      return;
+    if (maxChunksBeingTransferred < Long.MAX_VALUE) {
```

Comment:
Do we need this? `AtomicLong` seems efficient enough to always call `chunksBeingTransferred`

Reply:
> Do we need this? `AtomicLong` seems efficient enough to always call `chunksBeingTransferred`

OK,  how do you think about consistency and numerical convergence https://github.com/apache/spark/pull/30139#issuecomment-718528130

Reply:
If the concern is we may miss some places that update the state, then maybe only keeping this change is safer.

Reply:
> If the concern is we may miss some places that update the state, then maybe only keeping this change is safer.

OK, so I will just revert change about  `if (maxChunksBeingTransferred < Long.MAX_VALUE) {`

Reply:
Shouldn't we revert the change of `AtomicLong` and only keep `if (maxChunksBeingTransferred < Long.MAX_VALUE)`?

Reply:
> Shouldn't we revert the change of `AtomicLong` and only keep `if (maxChunksBeingTransferred < Long.MAX_VALUE)`?

All right,  I didn't get your point of comment https://github.com/apache/spark/pull/30139/files#r519647116, update  now

Reply:
ping @cloud-fan Any more suggestion?